docs/moderate.md

```diff
@@ -0,0 +1,199 @@
+# Moderate
+
+Let's say you want to moderate comments using OpenAI. You can do it in two ways:
+
+1. Using a trigger that will moderate the comment before it's inserted or updated in the database.
+2. Using a background action that will moderate the comments every [N configurable] seconds.
+
+## Via trigger
+
+You can get the full example in the [trigger_moderate.sql](examples/trigger_moderate.sql) file.
```

Comment:
```suggestion
```

That file doesn't exist anymore.

Reply:
Good point, I added it back just to avoid folks to suffer copying all fragments one by  one.

Reply:
Weird that it's not showing here but I added on f22a417d7caf4c894b37332bb5c706c6c150ba60